composer.json

```diff
@@ -16,10 +16,10 @@
         "doctrine/orm": "^2.6",
         "dragonmantank/cron-expression": "^2.3",
         "psr/log": "^1.1",
-        "symfony/console": "^4.3",
-        "symfony/framework-bundle": "^4.3",
-        "symfony/http-kernel": "^4.3",
-        "symfony/lock": "^4.3"
+        "symfony/console": "^4.3 || ^5.0",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/http-kernel": "^4.3 || ^5.0",
+        "symfony/lock": "^4.3 || ^5.0"
```

Comment:
As `LockFactory` didn't exist in `4.3` (see https://github.com/symfony/symfony/commit/fc75eb9bef8375fd0825f229046d987636d1aa07#diff-a367c9a860a7b4e43f7847078ef876fa), we should just bump everything to `^4.4 || ^5.0`